core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/stacks/request/cluster/ClusterV4Request.java

```diff
@@ -78,6 +78,9 @@
     @ApiModelProperty(ClusterModelDescription.VALIDATE_BLUEPRINT)
     private Boolean validateBlueprint = Boolean.TRUE;
 
+    @ApiModelProperty(StackModelDescription.CUSTOM_CONFIGS)
+    private String customConfigsName;
```

Comment:
is it possible to attach more than one?
customConfigsName -> customConfigurationNames ?